games/maze/src/maze.rs

```diff
@@ -83,10 +91,18 @@ impl Maze {
         self.map[i] = cell;
     }
 
-    fn get(&mut self, x: usize, y: usize) -> Tile {
+    fn get(&self, x: usize, y: usize) -> Tile {
         self.map[self.to_index(x, y)]
     }
 
+    fn get_tile_type(&self, x: i32, y: i32) -> TileType {
```

Comment:
A better name for this would be `tile_type_at(x, y)`. Really the get above should be `tile_at(x, y)` as well but don't worry about that one in this merge request.